twilight-lavalink/src/http.rs

```diff
@@ -258,7 +249,32 @@ pub fn load_track(
 ) -> Result<Request<&'static [u8]>, HttpError> {
     let identifier =
         percent_encoding::percent_encode(identifier.as_ref().as_bytes(), NON_ALPHANUMERIC);
-    let url = format!("http://{address}/loadtracks?identifier={identifier}");
+    let url = format!("http://{address}/v4/loadtracks?identifier={identifier}");
+
+    let mut req = Request::get(url);
+
+    let auth_value = HeaderValue::from_str(authorization.as_ref())?;
+    req = req.header(AUTHORIZATION, auth_value);
+
+    req.body(b"")
+}
+
+/// Decode a single track into its info
+///
+/// The response will include a body which can be deserialized into a
+/// [`Track`].
+///
+/// # Errors
+///
+/// See the documentation for [`http::Error`].
+pub fn decode_track(
```

Comment:
Function is called `decode_track` yet the request query has `encodedTrack`? This is confusing and/or wrong.